whylog/__init__.py

```diff
@@ -0,0 +1,14 @@
+from whylog.config import LineSource
+from whylog.config.filename_matchers import RegexFilenameMatcher
+from whylog.config.log_type import LogType
+from whylog.front.utils import FrontInput
+from whylog.front.whylog_factory import whylog_factory
+from whylog.log_reader import LogReader
+from whylog.teacher import Teacher
+assert FrontInput
```

Comment:
But pyflakes8 will not make noise of these lines.
Check: 
http://stackoverflow.com/questions/5033727/how-do-i-get-pyflakes-to-ignore-a-statement
http://www.laurivan.com/make-pyflakespylint-ignore-unused-imports/
I don't know the better solution. If you have better solution give it here.